app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/customView/AddPropertyInformationCustomView.java

```diff
@@ -43,13 +44,21 @@ private void initView(TypedArray attributes) {
     }
 
 
-    public static void setValueText( View view,String text) {
```

Comment:
Method does not need to be static
editText is a global variable inside the customView